infogami/infobase/logreader.py

```diff
@@ -1,19 +1,15 @@
 """
 Log file reader.
 """
-from __future__ import print_function
+
 import os
 import itertools
 import datetime
 import time
 import web
 import glob
 
-try:
-    import _json as simplejson
-except ImportError:
-    # make sure this module can be used indepent of infobase.
-    import simplejson
+import simplejson
```

Comment:
import json ??

Reply:
I got rid of our private fork of simplejson, so stuck with that for consistency rather than switching to a different module.

Reply:
json is builtin to Python.